stl/inc/xstring

```diff
@@ -2471,7 +2471,7 @@ private:
     static constexpr size_t _Memcpy_val_size   = sizeof(_Scary_val) - _Memcpy_val_offset;
 
     template <class _Iter>
-    using _Is_elem_cptr = bool_constant<_Is_any_of_v<_Iter, const _Elem* const, _Elem* const, const _Elem*, _Elem*>>;
+    static constexpr bool _Is_elem_cptr = _Is_any_of_v<_Iter, const _Elem* const, _Elem* const, const _Elem*, _Elem*>;

```

Comment:
Conventionally, variable templates are suffixed with `_v`

I believe we should rename this to `_Is_elem_cptr_v`

Reply:
> Conventionally, variable templates are suffixed with `_v`

Not in this library.
 
> I believe we should rename this to `_Is_elem_cptr_v`

No thank you.

Reply:
(All the other kids in the hall circle around @miscco and @CaseyCarter, as the chanting begins: "Fight, fight, fight!")